addons/purchase_requisition/models/purchase_requisition.py

```diff
@@ -421,10 +421,11 @@ class ProductProduct(models.Model):
     _inherit = 'product.product'
 
     def _prepare_sellers(self, params):
+        sellers = super(ProductProduct, self)._prepare_sellers(params)
```

Comment:
you can just write `super()` now! :smiley:

Reply:
@Elkasitu Really? Every PR I did with super() gots rejected because it isn't backportable/2.7 compatible. Have you changed policy at last.

Reply:
* For 11.0, not OK as we still support py27
* For 12.0, OK as we officially dropped support for py27 and support py35+ only [1]

[1] https://github.com/odoo/odoo/blob/12.0/odoo/__init__.py#L13-L15

Reply:
@Elkasitu Given there is precisely nowhere in the codebases that 11.0 and 12.0 share that super() is used, and this should be backported, I don't really think this is the patch to start using super().